packages/contracts/src/contracts/PublicLock/IPublicLockV13.sol

```diff
@@ -6,7 +6,7 @@ pragma experimental ABIEncoderV2;
  * @title The PublicLock Interface
  */
 
-interface IPublicLock {
+interface IPublicLockV13 {
```

Comment:
We need to number the interfaces for the docs to be parsed properly. That still manual step so should be automatized to avoid errors

Reply:
filed in #11633